apps/mochi-web/tailwind.config.js

```diff
@@ -4,9 +4,9 @@ import { consolelabs } from '@consolelabs/theme'
 module.exports = {
   content: [
     './**/*.{html,ts,tsx}',
-    './node_modules/@consolelabs/theme/dist/components/**/*.{js,jsx,ts,tsx}',
+    '!node_modules',
+    './node_modules/@consolelabs/theme/src/components/**/*.{ts,tsx}',
   ],
-  mode: 'jit',
```

Comment:
Can safely remove this, reference: https://tailwindcss.com/docs/upgrade-guide#migrating-to-the-jit-engine